Prace_domowe/Praca_domowa1/Grupa1/Dawid_Janus/PD1_Dawid_Janus.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>We wszystkich powyższych wizualizacjach/analizach brakuje komentarza. Na tych zajęciach umiejętność wnioskowania jest bardzo ważna.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>